packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -185,48 +198,47 @@ class _CircularProgressIndicatorPainter extends CustomPainter {
   static const double _kSweep = _kTwoPI - _kEpsilon;
   static const double _kStartAngle = -math.PI / 2.0;
 
-  const _CircularProgressIndicatorPainter({
+  _CircularProgressIndicatorPainter({
     this.valueColor,
     this.value,
     this.headValue,
     this.tailValue,
     this.stepValue,
-    this.rotationValue
-  });
+    this.rotationValue,
+    this.strokeWidth
+  }) {
+    if (value != null) { // Determinate progress indicator.
```

Comment:
The initial values of arcStart and arcSweep depend on this.value so they can't be initialized that way.